expression/expression.go

```diff
@@ -177,7 +177,15 @@ func evalExprToReal(expr Expression, row []types.Datum, sc *variable.StatementCo
 		return res, val.IsNull(), errors.Trace(err)
 	}
 	if expr.GetTypeClass() == types.ClassReal {
-		return val.GetFloat64(), false, nil
+		res, err = val.ToFloat64(sc)
+		return res, false, errors.Trace(err)
+		// TODO: We maintain two sets of type systems, one for Expression, one for Datum.
```

Comment:
can we simplify the TODO message to one single line? so that when we `grep "TODO"` we can get the message we want.

maybe: "TODO: remove this after refactoring all expressions" ?